### PR TITLE
Update imx500 notebooks for ci testing

### PR DESCRIPTION
**Pull request by Idan-BenAmi**
## Pull Request Description:
Update imx500 notebooks for CI testing - a waiver on GPTQ sections and waver on Imagenet download

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/resources/utils/pytorch_tutorial_tools.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
```
Comment (reuvenperetz) [inline tutorials/resources/utils/pytorch_tutorial_tools.py:1]: year..

File: tutorials/notebooks/imx500_notebooks/keras/keras_yolov8n_for_imx500.ipynb [hunk 2]
```diff
@@ -278,6 +278,9 @@
     "representative_dataset_gen = get_representative_dataset(n_iters, rep_data_loader)"
    ],
    "metadata": {
+    "tags": [
+     "remove_cell"
```
Comment (reuvenperetz) [inline tutorials/notebooks/imx500_notebooks/keras/keras_yolov8n_for_imx500.ipynb:282]: Maybe use other tag like "long run" or something...

File: tutorials/notebooks/imx500_notebooks/pytorch/pytorch_yolov8n_for_imx500.ipynb [hunk 3]
```diff
@@ -302,6 +302,9 @@
     "                                                        dataset_loader=gptq_representative_dataset)"
    ],
    "metadata": {
+    "tags": [
+     "remove_cell"
```
Comment (reuvenperetz) [inline tutorials/notebooks/imx500_notebooks/pytorch/pytorch_yolov8n_for_imx500.ipynb:306]: Same..

**Review by reuvenperetz — APPROVED** [review]
